packages/@aws-cdk/aws-redshift-alpha/lib/cluster.ts

```diff
@@ -558,6 +565,17 @@ export class Cluster extends ClusterBase {
       );
     }
 
+    const nodeType = props.nodeType || NodeType.DC2_LARGE;
```

Comment:
do we need to specify it here its already set as the default for this prop?

Reply:
This specification is not new.

https://github.com/aws/aws-cdk/pull/29976/files#diff-8666885f385d3c87fb98f835d762b86e7a98bfd217bd69c9a210041ab07585d6L577

If we change this implementation, it will cause a breaking change. Therefore, I think it is better to keep it as is.

Reply:
understand, I should've been more specific wanted to check if we could keep it as `const nodeType = props.nodeType` , since NodeType.DC2_LARGE is already set as default but I see that we've kept it as before, thank you for explaning.